indra/newview/llinventorymodel.cpp

```diff
@@ -3904,7 +3904,6 @@ void LLInventoryModel::processBulkUpdateInventory(LLMessageSystem* msg, void**)
 			cblist.push_back(InventoryCallbackInfo(callback_id, LLUUID::null));
 		}
 	}
-	gInventory.accountForUpdate(update);
```

Comment:
This is wrong.
1. 'update' in such case is not needed
2. This is a message by server that covers a lot of cases, not just moving items/folders, it's likely to break something.
3. In viewer-server relationship server is the one we trust, viewer should not pre-increment version if it knows server will respond